py/torch_tensorrt/fx/converters/converter_utils.py

```diff
@@ -122,7 +122,12 @@ def set_layer_name(
         if isinstance(target, str)
         else f"{source_ir}_ops.{target.__name__}"
     )
-    layer.name = f"[{layer.type.name}]-[{target_name}]-[{name}]"
+    layer.name = f"[{layer.type.name}]-[{target_name}]-[{name}]-[#_of_outputs_{layer.num_outputs}]"
+
+    for i in range(layer.num_outputs):
+        output = layer.get_output(i)
+        layer.name.append(f"-[{output.name}]")
```

Comment:
Should `layer.name.append(f"-[{output.name}]")` instead be `layer.name += f"-[{output.name}]"`?

Reply:
We should not just append all the output tensor names to layer name because that makes layer name very long